src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const populationElements = document.querySelectorAll('.population');
+
+  const populationNumbers = Array.from(populationElements)
+    .map((el) => el.textContent.replace(/,/g, ''))
+    .filter((str) => !isNaN(str))
+    .map(Number);
+
+  const totalPopulation = populationNumbers.reduce((sum, num) => sum + num, 0);
+  const averagePopulation = totalPopulation / populationNumbers.length;
```

Comment:
If there are no elements with the class 'population', `populationNumbers.length` will be zero, leading to a division by zero when calculating `averagePopulation`. Consider adding a check to handle this case to avoid potential errors.